src/v0/destinations/customerio/util.js

```diff
@@ -51,6 +52,29 @@ const getSizeInBytes = (obj) => {
   return bytes;
 };
 
+const convertTimestampsToUnix = (payload) => {
```

Comment:
this function is heavy, do we really need to iterate all fields and check if it is timestamp field and apply this logic? can't be do on specific fields?
can you attach documentation on why such requirement exists?